Dockerfile

```diff
@@ -8,4 +8,15 @@ WORKDIR /portus
 COPY Gemfile* ./
 RUN bundle install --retry=3
 
+# Install phantomjs, this is required for testing and development purposes
+# There are no official deb packages for it, hence we built it inside of the
+# open build service.
+RUN echo "deb http://download.opensuse.org/repositories/home:/flavio_castelli:/phantomjs/Debian_8.0/ ./" >> /etc/apt/sources.list
+RUN wget http://download.opensuse.org/repositories/home:/flavio_castelli:/phantomjs/Debian_8.0/Release.key && \
+  apt-key add Release.key && \
+  rm Release.key
+RUN apt-get update && \
+    apt-get install -y phantomjs && \
+    apt-get clean
```

Comment:
Just a small nitpick. I've seen in jfrazelle/dockerfiles that instead of `apt-get clean` she does:

```
	&& rm -rf /var/lib/apt/lists/*
```

I don't know which is best. Moreover, I'd add `--no-install-recommends` to the install command.